cmd/ako-main/main.go

```diff
@@ -78,13 +81,21 @@ func InitializeAKC() {
 
 	var crdClient *crd.Clientset
 	var advl4Client *advl4.Clientset
+	var svcAPIClient *svcapi.Clientset
 	if lib.GetAdvancedL4() {
 		advl4Client, err = advl4.NewForConfig(cfg)
 		if err != nil {
-			utils.AviLog.Fatalf("Error building service-api clientset: %s", err.Error())
+			utils.AviLog.Fatalf("Error building service-api v1alpha1pre1 clientset: %s", err.Error())
 		}
 		lib.SetAdvL4Clientset(advl4Client)
 	} else {
+		if lib.UseServicesAPI() {
```

Comment:
Can advancedL4 and servicesAPI be both enabled?Reason being that , else can be ommited.